modules/light-clients/08-wasm/types/vm.go

```diff
@@ -26,16 +26,16 @@ var (
 	// wasmvmAPI is a wasmvm.GoAPI implementation that is passed to the wasmvm, it
 	// doesn't implement any functionality, directly returning an error.
 	wasmvmAPI = wasmvm.GoAPI{
-		HumanAddress:     humanAddress,
-		CanonicalAddress: canonicalAddress,
+		HumanizeAddress:     humanizeAddress,
+		CanonicalizeAddress: canonicalizeAddress,
```

Comment:
Here you need a validate implementation too now. Can be the same style as the other two but needs to be set.